Lib/test/test_curses.py

```diff
@@ -1319,5 +1320,75 @@ def lorem_ipsum(win):
     for y, line in enumerate(text[:maxy]):
         win.addstr(y, 0, line[:maxx - (y == maxy - 1)])
 
+
+class TextboxTest(unittest.TestCase):
+    def setUp(self):
+        self.mock_win = unittest.mock.MagicMock(spec=curses.window)
```

Comment:
The import on line 9 is unused because you're using the fq path here.

```suggestion
        self.mock_win = MagicMock(spec=curses.window)
```